### ISSUE TITLE
Improve color contrast for concept badge

### ISSUE THREAD

**Issue by drkrynstrng** [jantman/repostatus.org#31]
The concept badge has poor color contrast which makes it difficult to read the word "Concept" in the badge. To improve the contrast, one option is changing the text color to black or another is changing the background to a darker color (maybe black?).

**Comment by jantman** [comment]
@drkrynstrng I haven't done much work on this project lately... and I'm really not terribly knowledgeable about the best/most accessible contrast schemes. PRs - or specific suggestions - welcome though!

**Comment by polynoman** [comment]
@drkrynstrng please check out my pull request. Does it fix this issue for you?  @jantman Pleace check out if you are happy with my proposal.

**Comment by drkrynstrng** [comment]
Yeah, that's more readable. Now that I'm seeing it, I recommend a shade of gray for the text color instead, like #666666. This would make it slightly less dark and differentiate the badge text a little from surrounding header and body text.

**Comment by polynoman** [comment]
@drkrynstrng I followed the official contribution way as @jantman explained me. It seems like the badge that comes from shields.io now has a dark text color for the white background by default. I have updated my pull request with a new version 2.1.0, please check it out.

**Comment by drkrynstrng** [comment]
Looks good to me. Thanks.

**Comment by jantman** [comment]
This has been fixed in #35 and released. Thanks, and apology for the delay.
